frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/mlmdUtils.ts

```diff
@@ -47,3 +49,8 @@ export const initMlmdIntercepts = (
     mockGetEventsByExecutionIDs(),
   );
 };
+
+export const decodeMlmdRequests = (interception: Interception): GetExecutionsRequest => {
```

Comment:
This is a very generic function name. Isn't this specific to decoding `GetExecutionRequest`  and therefore should be named as such?

Perhaps parameterize to accept `GetExecutionsRequest` so it can be generic? or otherwise we'll be duplicating this function for each decoding we do.